### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by coffeecup-winner**
- Don't stop server on keypress
- Enable logging in tests
- Change the code to be more functional

### CODE REVIEW

File: src/KawaBot.Tests/BasicControllerTests.fs [hunk 1]
```diff
@@ -16,12 +19,15 @@ type BasicControllerTests =
             _server = new Http.HttpListener("http://*:80/")
             _feeder = new Feeder(80)
         } then
+            let config = new LoggingConfiguration()
+            config.AddRule(LogLevel.Trace, LogLevel.Fatal, new ConsoleTarget(), "*")
+            LogManager.Configuration <- config
```
Comment (sirIrishman) [inline src/KawaBot.Tests/BasicControllerTests.fs]: Can't we put this configuration into the `App.config` file? And moreover with applying for all loggers?

File: src/KawaBot.Server/Utils.fs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿namespace KawaBot.Server
+
+open System.Collections.Generic
+
+[<AutoOpen>]
+module Utils =
+    let tryGet (dict: IReadOnlyDictionary<'K, 'V>) (key: 'K): option<'V> =
```
Comment (sirIrishman) [inline src/KawaBot.Server/Utils.fs]: Have we already discussed name convention? Why `tryGet`, but not `TryGet`?

File: src/KawaBot.Tests/BasicControllerTests.fs [hunk 3]
```diff
@@ -16,13 +19,16 @@ type BasicControllerTests =
         {
             _server = new Http.HttpListener("localhost", 49999)
         } then
+            let config = new LoggingConfiguration()
+            config.AddRule(LogLevel.Trace, LogLevel.Fatal, new ConsoleTarget(), "*")
+            LogManager.Configuration <- config
```
Comment (sirIrishman) [inline src/KawaBot.Tests/BasicControllerTests.fs]: Can't we put this configuration into the `App.config` file?

File: src/KawaBot.Server/Utils.fs (hunk 2 above)
Comment (coffeecup-winner) [inline src/KawaBot.Server/Utils.fs]: consistency with function names in standard lib modules, like Option.map or Seq.unfold

File: src/KawaBot.Tests/BasicControllerTests.fs (hunk 3 above)
Comment (coffeecup-winner) [inline src/KawaBot.Tests/BasicControllerTests.fs]: done

**Pull request merged**